.github/workflows/build_test.yaml

```diff
@@ -0,0 +1,218 @@
+name: Build and test
+
+on:
+  pull_request:
+    branches:
+      - main
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: RelWithDebInfo
```

Comment:
I think we should build in `Release` mode, because the benefit of `RelWithDebInfo` is to just have the debug symbols and we never use it.

Reply:
Sure, I am fine with Release mode.

Reply:
Ok.